src/hooks/usePortalContainer/README.md

```diff
@@ -0,0 +1,17 @@
+<!--GITHUB_BLOCK-->
+
+# usePortalContainer
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {usePortalContainer} from '@gravity-ui/uikit';
+```
+
+The `usePortalContainer` hook helps to create portal container
```

Comment:
This hook doesn't create a portal container, it returns portal container element in the current scope